frontend/mulighetsrommet-cms/deskStructures/deskStrukture.ts

```diff
@@ -10,11 +10,6 @@ export const structure = (S, context) => {
 
   const deskItems = [];
 
-  if (roleNames.includes("administrator")) {
```

Comment:
Flyttet denne nederst i funksjonen siden default oppførsel uansett er å vise `adminStructure`. Dette gjør også at deskstructure for rollene `redaktor_av_dir` og `redaktor-tiltaksgjennomforing` får høyere pri enn adminstrucure.